connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -397,6 +398,42 @@ object KustoWriter {
     writer.out.toString
   }
 
+  private def writeDecimalField(row: SpecializedGetters, fieldIndexInRow: Int, precision: Int,scale: Int, writer: Writer) = {
```

Comment:
What about moving this method (and the other write* method to another class? (Or it can be like an extension method in C#). Will make this file smaller

Reply:
Tried with extension methods over SpecializedGetters but it really made it complex ,
ill take your other suggestion and put in another class